### PR TITLE
158 get method not returning all items

### PR DESCRIPTION
**Pull request by mikeerickson**
- [ ] Fixed issue when using `validator.errors.all()` and `validator.errors.get('attr')` when defining multiple rules (such as `min:10|emal`)   This does not affect issue when `required` is supplied.
- [ ] Added new test to verify this condition
- [ ] Ran full test suite (`npm test`) to assure all builds and validates

Fixes https://github.com/skaterdav85/validatorjs/issues/158

### CODE REVIEW

File: spec/error-messages.js [hunk 1]
```diff
@@ -192,8 +192,24 @@ describe('Error messages', function() {
...
+      }, {
+        email: 'email|min:10'
+      });
+
+      expect(validator.passes()).to.be.false;
+      expect(validator.errors.get('email')).to.be.instanceOf(Array);
+      expect(validator.errors.get('email').length).to.equal(2);
+
+      expect(validator.errors.all()).to.be.instanceOf(Object);
+      expect(validator.errors.get('email').length).to.equal(2);
```
Comment (iamdtang) [inline spec/error-messages.js]: duplicate assertion?

**Review by iamdtang — CHANGES_REQUESTED** [review]

**Review by iamdtang — APPROVED** [review]
looks good to me. @garygreen ?

**Comment by mikeerickson** [comment]
@garygreen hey bud, just checking in.  Did you have any further requests?

**Comment by garygreen** [comment]
Looks good to me too

**Pull request merged**